GetIntoTeachingApiTests/Services/CrmServiceTests.cs

```diff
@@ -343,8 +347,8 @@ public void MatchCandidate_WithMasterChildRecords_ReturnsMasterRecord()
         {

```

Comment:
Just noticed these whitespace changes sorry, I can remove them if you want. I bash Ctrl+k Ctrl+D (not sure what the mac equivalent is) to format a lot and it does this.